include/library.h

```diff
@@ -264,16 +264,16 @@ class Library: public std::enable_shared_from_this<Library>
   bool removeBookmark(const std::string& zimId, const std::string& url);
 
   /**
-   * Resolve bookmarks
+   * Migrate bookmarks
    *
-   * @param source the source bookId of the bookmarks to resolve.
+   * @param source the source bookId of the bookmarks to migrate.
                    If not present, we resolve all bookmarks pointing to an unknown book.
-   * @param destination the destination bookId to resolve te bookmark to.
+   * @param destination the destination bookId to migrate the bookmarks to.
                         If not present, we will try to determine it using heuristics based on
                         book name, flavour and date.
-   * @return The number of bookmarks created.
+   * @return The number of bookmarks updated.
    */
-  int resolveBookmarks(const std::string& source = "", std::string destination = "");
+  int migrateBookmarks(const std::string& source = "", std::string destination = "");
```

Comment:
Reminding that during rebase&squash the commit message of the original commit must be updated too